backend/src/server/routes/v1/organization-router.ts

```diff
@@ -29,18 +29,30 @@ export const registerOrgRouter = async (server: FastifyZodProvider) => {
     },
     schema: {
       response: {
-        200: z.object({
-          organizations: sanitizedOrganizationSchema
-            .extend({
-              orgAuthMethod: z.string(),
-              userRole: z.string()
-            })
-            .array()
-        })
+        200: z.union([
+          z.object({
+            id: z.string(),
+            name: z.string(),
+            slug: z.string(),
+            role: z.string()
+          }),
```

Comment:
why do we return a completely different schema for identities? 

Can we unify it to use `sanitizedOrganizationSchema` without the `.extend` section?